### ISSUE TITLE
Dark theme problem

### ISSUE THREAD

**Issue by baur** [namnguyenthanhwork/docusaurus-material-ui-template#4]
Thank you for your work. Everything functions as expected, but there are issues with the MJI components.

```
import Divider from "@mui/material/Divider";
import Chip from "@mui/material/Chip";
...
<Divider><Chip label={`Dark theme`}/>
...
```
Light

![image](https://github.com/user-attachments/assets/efd092db-2386-49fd-82e7-7a55075c6fca)

Dark

![image](https://github.com/user-attachments/assets/73dffdfe-6ab5-4bec-a7b2-3a4d6cc4a695)

**Comment by baur** [comment]
with MUI v5 worked well

v5 > v6

![image](https://github.com/user-attachments/assets/c76df668-4fa6-473a-b409-86e90c50618b)

v5 > v6

![image](https://github.com/user-attachments/assets/d7bb8f40-e22e-4151-b8ab-cb0407833dda)

**Comment by namnguyenthanhwork** [comment]
Thanks, @baur! I fixed it at #5 . However, I think you should recheck the styling because this header uses a background image.

**Comment by baur** [comment]
Thanks,
It’s fixed!
